components/Slider/README.md

```diff
@@ -215,3 +219,31 @@ id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] init];
 ```
 <!--</div>-->
 
+<!-- Extracted from docs/accessibility.md -->
+
+## Accessibility
+To help ensure your slider is accessible to as many users as possible, please be sure to review the following recommendations:
+
+### Set `-accessibilityLabel`
+
+Set an appropriate `accessibilityLabel` value for your slider. This should reflect what the slider affects.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Objective - C
+```objc
+slider.accessibilityLabel = @"Volume Slider";
```

Comment:
Please [don't include the name of the control in the label](https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel?language=objc).  The `accessibilityLabel` could be something like "audio volume", or "volume level", or "headphone volume".

> Note, however, that the label should never include the control type (such as “button”) because this information is contained in the traits associated with the accessibility element.
> &mdash; https://developer.apple.com/documentation/objectivec/nsobject/1615181-accessibilitylabel?language=objc